dev/ledger.md

```diff
@@ -1400,18 +1515,20 @@ the block's round $r$ and for the block's genesis identifier $\GenesisID_B$.
 For a transaction
 $$\Tx = (\GenesisID, \TxType, r_1, r_2, I, I', I_0, f, a, x, N, \pk, \sppk, \nonpart,
   \ldots)$$
-(where $\ldots$ represents fields specific to asset transaction types)
+(where $\ldots$ represents fields specific to transaction types
+besides "pay" and "keyreg")
 to be valid at the intermediate state $\rho$ in round $r$ for the genesis
 identifier $\GenesisID_B$, the following conditions must all hold:
 
  - It must represent a transition between two valid account states.
  - Either $\GenesisID = \GenesisID_B$ or $\GenesisID$ is the empty string.
- - $\TxType$ is either "pay", "keyreg", "acfg", "axfer", "afrz" or "appl".
+ - $\TxType$ is either "pay", "keyreg", "acfg", "axfer", "afrz",
+   "appl", or "hb".
```

Comment:
It is not strictly related to this PR, but I think the State Proof transaction type is missing here.

Reply:
Addressed in #107